package.json

```diff
@@ -32,20 +32,19 @@
   },
   "homepage": "https://github.com/emiljanitzek/mongoose-delete-ts",
   "peerDependencies": {
-    "mongoose": "6.x"
+    "mongoose": "8.x"
   },
   "devDependencies": {
-    "@tsconfig/node18": "~1.0.1",
-    "@types/chai": "~4.3.1",
-    "@types/mocha": "~9.1.1",
-    "@typescript-eslint/eslint-plugin": "~5.30.6",
-    "@typescript-eslint/parser": "~5.30.6",
-    "chai": "~4.3.6",
```

Comment:
I was not able to update `chai` to the latest version (5.0.0).
Test is not working with `chai` 5.0.0.

```
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /home/achilles/Work/mongoose-delete-ts/test/aggregation.test.ts
    at new NodeError (node:internal/errors:399:5)
    at Object.getFileProtocolModuleFormat [as file:] (node:internal/modules/esm/get_format:79:11)
    at defaultGetFormat (node:internal/modules/esm/get_format:121:38)
    at defaultLoad (node:internal/modules/esm/load:81:20)
    at nextLoad (node:internal/modules/esm/loader:163:28)
    at ESMLoader.load (node:internal/modules/esm/loader:605:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:457:22)
    at new ModuleJob (node:internal/modules/esm/module_job:64:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:480:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:434:34)
```